docs/multi-agent-manager.md

```diff
@@ -6,13 +6,21 @@ The <a href="../api/manager/manager">`MultiAgentManager`</a> allows managing mul
 We instantiate the manager by providing it with the working directory in which to operate and starting it:
 
 ``` python
+import os
 from pathlib import Path
 from aea.manager import MultiAgentManager
 
-WORKING_DIR = "."
+WORKING_DIR = "mam"
+ALIASES_DIR = "aliases"
+KEYS_DIR = str(Path(WORKING_DIR, ALIASES_DIR, "keys"))
+CERTS_DIR = str(Path(WORKING_DIR, ALIASES_DIR, "certs"))
+BUILD_DIR = str(Path(WORKING_DIR, ALIASES_DIR, "build"))
```

Comment:
Ideally this would all be done by the manager, but atm we need to manually do it.